### PR TITLE
catch invalid_parameter in register_trace()

### PR DESCRIPTION
**Pull request by zeze-zeze**
As [issue #157](https://github.com/microsoft/krabsetw/issues/157) said, there is an error when manage the provider on Microsoft-Windows-Security-Auditing, tested on Windows10 1709, Windows server 2012 R2.

So I handle error code invalid_parameter (87) just the same as the catch for need_to_be_admin_failure (5).

### CODE REVIEW

File: krabs/krabs/etw.hpp
Comment (swannman) [inline krabs/krabs/etw.hpp:309]: could you add a comment on the line after `catch` describing the scenario that you are addressing with this change, so it's clear to future maintainers of this code?

**Review by swannman — CHANGES_REQUESTED** [review]
Thanks for submitting this! I added one minor comment to your change. Could you also bump the minor version number in the three .nuspec files at the project root so we can publish the updated nugets after merging your change?

**Comment by zeze-zeze** [comment]
Done. I have commented on the line after catch and changed the version of .nuspec files.

**Review by swannman — APPROVED** [review]

**Pull request merged**
